ssl/ssl_lib.cc

```diff
@@ -1995,19 +1995,21 @@ const char *SSL_get_cipher_list(const SSL *ssl, int n) {
 }
 
 int SSL_CTX_set_cipher_list(SSL_CTX *ctx, const char *str) {
-  return ssl_create_cipher_list(&ctx->cipher_list, str, false /* not strict */);
+  return ssl_create_cipher_list(&ctx->cipher_list, str, false /* not strict */,
+                                false /* not tls1.3 only */);
```

Comment:
I understand that the reasoning behind `not tls1.3 only` means "not `tls1.3_only`", but it can sort of be misconstrued as "not only for tls1.3". Maybe "not for tls1.3" might be better?

Reply:
How about rename `tls13_only` to `config_tls13`?
```cc
`true`:  /* configure TLSv1.3 ciphers */
`false`: /* not configure TLSv1.3 ciphers */
```

Reply:
nit:
```
`false`: /* don't configure TLSv1.3 ciphers */
```